src/containers/Tenant/Diagnostics/Healthcheck/Details/Details.tsx

```diff
@@ -6,30 +6,30 @@ import updateArrow from '../../../../../assets/icons/update-arrow.svg';
 
 import type {IIssuesTree} from '../../../../../types/store/healthcheck';
 
-import IssuesViewer from '../IssuesViewer/IssuesViewer';
+import IssueTree from '../IssuesViewer/IssueTree';
 
 import i18n from '../i18n';
 
 const b = cn('healthcheck');
 
-interface IssuesListProps {
-    issue?: IIssuesTree;
+interface IssuesDetailsProps {
```

Comment:
that's healthcheck details, not issues details
and usually it is based on the component naming, so `DetailsProps`